status-go-version.json

```diff
@@ -2,7 +2,7 @@
     "_comment": "DO NOT EDIT THIS FILE BY HAND. USE 'scripts/update-status-go.sh <tag>' instead",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "5f2544f1e8f2ee832bbab95c8d74e26ba0ab2481",
-    "commit-sha1": "5f2544f1e8f2ee832bbab95c8d74e26ba0ab2481",
-    "src-sha256": "0dlkr6056mhyny9k895zs5012ywiz75qm6y7a6hypb3pfxr35931"
+    "version": "develop",
```

Comment:
Please don't use `develop` as a version for `status-go`. That will cause the version to appear in the App (and for the p2p peer) as `develop`, which is meaningless. It's fine for development, but it's not fine for merging into `develop` in `status-react`.

I will be releasing `0.34.0-beta.4` today, I'll ping you when I do, please use that.

Reply:
cool, let me know when ready